accounts/admin.py

```diff
@@ -24,26 +24,41 @@ def last_name(self, obj):
 
 
 class UserAdmin(admin.ModelAdmin):
-    readonly_fields = ("username", "pennid", "last_login", "date_joined")
+    readonly_fields = ("username", "first_name", "last_name", "pennid", "last_login", "date_joined")
     search_fields = ("username", "first_name", "last_name")
     list_display = ("username", "first_name", "last_name", "is_staff")
     list_filter = ("is_staff", "is_superuser", "is_active", "groups")
     filter_horizontal = ("groups", "user_permissions")
     ordering = ("username",)
     fieldsets = (
         (None, {"fields": ("username", "pennid")}),
-        (("Personal info"), {"fields": ("first_name", "last_name", "email")}),
+        (("Personal info"), {"fields": ("preferred_name", "email")},),
```

Comment:
I think we should probably undo the changes to readonly_fields and add first/last name back to the personal info section just for the sake of an easier experience in local dev